recursive_monkey_patch.py

```diff
@@ -206,6 +208,14 @@ def monkey_patch(source, target, log_level=logging.WARNING, logger=None):
 
         >>> A.NestedNew.__module__
         'a_test_module_patch.submodule'
+
+    .. RUBRIC:: Testing that new modules behave like old modules
+
+    The new modules can be imported (actually they are already imported)::
+
+        >>> import a_test_module.submodule_new
+        >>> from a_test_module.submodule_new import i
```

Comment:
What about displaying the value of `a_test_module.submodule.__module__`.

Reply:
Hm, `AttributeError: 'module' object has no attribute '__module__'`